lib/rubocop/cop/style/symbol_array.rb

```diff
@@ -57,13 +66,24 @@ def on_array(node)
 
         private
 
-        def symbols_contain_spaces?(node)
+        def complex_content?(node)
+          delimiters = ['[', ']', '(', ')']
```

Comment:
@jasondoc3 It could be a constant instead of a variable :-)

Reply:
Done!